cads_adaptors/adaptors/__init__.py

```diff
@@ -141,14 +141,53 @@ def apply_constraints(self, request: Request) -> dict[str, Any]:
 
     @abc.abstractmethod
     def estimate_costs(self, request: Request) -> dict[str, int]:
+        """
+        Estimate the costs associated with the request.
+
+        Parameters
+        ----------
+        request : Request
+            Incoming request.
+
+        Returns
+        -------
+        dict[str, int]
+            Estimated costs.
```

Comment:
```suggestion
            Estimated costs, where the key is the cost name/type (e.g. size/precise_size) and the value is the maximum cost for that type.
```

Reply:
something to this effect could be added to the description if you prefer. Essentially, other people need to know what the returned dictionary shuld contain

Reply:
sounds good. I'll add the same wording in the confluence page